hack/prepare.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -xe
+
+source hack/config
+
+# Install the latest version of Docker
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+sudo apt-get update
+sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
+
+# Enable experimental features for the Docker client
+mkdir -p ~/.docker
+cp hack/docker-client.json ~/.docker/config.json
+
+# Enable experimental features for the Docker daemon
+sudo cp hack/docker-daemon.json /etc/docker/daemon.json
+sudo systemctl restart docker
+
+# Install the buildx Docker plugin
+DOCKER_BUILDKIT=1 docker build --platform=local -o . git://github.com/docker/buildx
+mkdir -p ~/.docker/cli-plugins
+mv buildx ~/.docker/cli-plugins/docker-buildx
+
+# Set up the cross-architecture build environment
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
I understand this line but I don't understand how builds can work afterwards.

Are you somewhere copying the qemu-static binary into the container? If not, how can e.g. the arm64 container reach the corresponding qemu-static?

Reply:
Basically when `-p yes` is used the qemu-static binaries are loaded into memory and no longer need to be accessed from the filesystem, which means that any container from any architecture can be used transparently from that moment forward. See https://github.com/multiarch/qemu-user-static#multiarch-compatible-images-deprecated.

Reply:
Nice. I missed that flag.

Reply:
@rmohr anything else I need to do to get this in?